addons/sourcemod/scripting/vsh/bosses/boss_uberranger.sp

```diff
@@ -360,16 +343,64 @@ methodmap CMinionRanger < SaxtonHaleBase
 			SetEntProp(iWearable, Prop_Send, "m_nModelIndexOverrides", g_iUberRangerBlightedBeak);
 			SetEntityRenderColor(iWearable, iColor[0], iColor[1], iColor[2], iColor[3]);
 		}
+		
+		g_hUberRangerMinionAFKTimer[this.iClient] = CreateTimer(0.0, Timer_UberRanger_ReplaceMinion, this.iClient);
```

Comment:
`g_hUberRangerMinionAFKTimer` should be set to null from `Destory` and/or `OnDeath`